class.jetpack.php

```diff
@@ -1096,16 +1093,6 @@ function get_locale() {
 		return $locale;
 	}
 
-	/**
-	 * Device Pixels support
-	 * This improves the resolution of gravatars and wordpress.com uploads on hi-res and zoomed browsers.
-	 */
-	function devicepx() {
```

Comment:
I couldn't think of reasons to keep it. The most typical way to remove `devicepx` seems to be `wp_dequeue_script( 'devicepx' );` so we should be good.